libwdi-sys/build.rs

```diff
@@ -341,6 +341,14 @@ impl LibwdiBuild
             self.apply_patch_file(src_file, patch_file);
         }
 
+        // Disable target support if flag set
+        if cfg!(feature = "disable-x86") {
+            self.replace_in_file(Path::new("msvc/config.h"), "#define OPT_M32", "//#define OPT_M32");
```

Comment:
This seems to me to be potentially brittle. Is this the only way to en-/dis-able the features?

But even if it is not, will this error if the entry is not found in the `config.h` file? I think we should be explicit when we fail to find the options in the configuration.